doc/learn_more.rst

```diff
@@ -16,10 +16,13 @@
 
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+.. _learn_more_ref:
 
 Learn More
 ==========
-We are working hard to expand this user guide, but if you are stuck and don't know where to turn,
-please submit a question or bug report here: Issues_.
+We are working hard to expand this user guide and to improve OpenShot Video Editor, but if you are stuck and don't know where to turn, OpenShot has several sources for additional information.
 
-.. _Issues: https://github.com/OpenShot/openshot-qt/issues/
+ #. OpenShot has several `YouTube videos <https://www.youtube.com/results?search_query=openshot+video+editor+jonathan+thomas>`_ available to help you see how to use the application.
+ #. OpenShot has a `Reddit Community <https://www.reddit.com/r/OpenShot/>`_ dedicated to users helping users with questions about the video editor and computer-related issues.
+ #. If Reddit and Youtube did not help, and you believe that you have a bug, you can visit the `GitHub repository <https://github.com/OpenShot/openshot-qt/issues>`_.
+ #. If all of the above did not help you, you can open a ticket by sending an E-mail message to the `OpenShot Support Team <mailto://support@openshot.org>`_.
```

Comment:
Maybe we should offer a paid support option as well? For high priority customers who want immediate responses and debugging?